app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntrySheet.kt

```diff
@@ -157,6 +182,28 @@ class BloodPressureEntrySheet : BottomSheetActivity() {
     }
   }
 
+  private fun screenTypeChanges(): Observable<UiEvent> {
+    // RxViewFlipper
+    //     .displayedChildChanges
+    //     .map()
+    return Observable.just(BloodPressureScreenChanged(ScreenType.BP_ENTRY))
```

Comment:
@rakshakhegde I moved the initial event of `BloodPressureScreenChanged(ScreenType.BP_ENTRY)` to this function. It's very common to emit initial events and it's in fact the job of each ui event function to ensure an initial event is emitted. 

If you take a look at RxBinding's source, you'll find that every observer emits a default value on subscription. This makes it easy to setup UI on start.